resources/main.css

```diff
@@ -287,15 +287,28 @@ button,
 }
 
 .developer-mode-content button {
+    font-size: 16px;
     background: white;
     flex: auto;
     padding: 4px;
     appearance: button;
     border: 2px solid rgba(255, 255, 255, 0.5);
     border-radius: 10px;
 }
+
+.developer-mode-content button span {
+    padding-left: 10px;
+    font-size: 25px;
```

Comment:
i'd use a size in em (for example `1.5em` looks fine)

Reply:
Done.